src/components/FormGroup/FormGroup.test.ts

```diff
@@ -10,6 +10,27 @@ describe('template', () => {
 
     componentSlotRenderTest(FormGroup);
 
+    it('renders scoped slot', async() => {
+        const wrapper = shallowMount(FormGroup, {
+            props: {
+                invalidFeedback: 'a',
+                validFeedback: 'a',
```

Comment:
This test is a bit confusing because in reality this scenario should ideally never happen.. Valid and invalid at the same time. I'd suggest to change the prop to prevent future confusion.

Reply:
I disagree. Why can't you have invalid and valid feedback at the same time?
![image](https://user-images.githubusercontent.com/45201651/143404270-8d917bdc-9144-4464-86cd-35596fa17494.png)
Changed this in BS documentation.